docker-compose.yml

```diff
@@ -29,9 +23,10 @@ services:
       DEPENDENCIES: "influxdb"
     labels:
       - "ServiceUUId=system.telegraf"
+      - "io.appcelerator.amp.build.mounts=/var/run/docker.sock,/var/run/utmp"
```

Comment:
you can do something like `-m type=bind,source=/vol/nginx,target=/etc/nginx,writable=false` to mount readonly